pipreqs/pipreqs.py

```diff
@@ -145,11 +178,35 @@ def get_all_imports(path, encoding=None, extra_ignore_dirs=None, follow_links=Tr
     return list(packages - data)
 
```

Comment:
```suggestion

def get_file_extensions():
    return [".py", ".ipynb"] if PythonExporter and not ignore_notebooks else [".py"]
```

Reply:
Decided to not implement this function because it makes the file extensions used by the program "hidden". The user will only know the file extension being used in the program if the user reads the function directly, making our code a little bit less readable